cmd/kube-scheduler/app/options/configfile_test.go

```diff
@@ -128,7 +129,7 @@ func TestLoadConfigFromFile(t *testing.T) {
 			if test.expectedErr == nil {
 				assert.NoError(t, err)
 			} else {
-				assert.ErrorContains(t, err, test.expectedErr.Error())
+				assert.ErrorIs(t, err, test.expectedErr)
```

Comment:
commented at the new error Is implementations as well, but this is relying on empty notregistered errors returning true for any other notregistered error, which I don't think is correct

if the goal of this change was to handle the `os.ErrNotExist` error generically, I'd suggest something like this instead, and dropping the Is implementations and string match changes above:

```go
if !errors.Is(err, test.expectedErr) && !strings.Contains(err.Error(), test.expectedErr.Error()) {
  t.Errorf("expected %v to contain %q or return true for Is(%#v)", err, test.expectedErr.Error(), test.expectedErr)
}
```

Reply:
Removed this for now, might submit some other time. This was meant to address some issues with the Winodws unit tests, but ``TestLoadConfigFromFile`` seems to pass on Windows.